knb

```diff
@@ -184,6 +186,8 @@ function usage {
 	    -t <time-in-scd>
 	    --timeout <time-in-scd>     : Set the pod ready wait timeout in seconds (Default ${POD_WAIT_TIMEOUT})
 
+	    --multi <multi>               : Run multiple instances of iperf3.(Default 1, Maximum 99)
```

Comment:
I'm not very found of `--multi` but I've never been good at assigning names :D let me know if you got any suggestions!